eiffel-vocabulary/EiffelServicePausedEvent.md

```diff
@@ -0,0 +1,20 @@
+
+# EiffelServicePausedEvent (ArtS)
+
+The EiffelServicePausedEvent declares that a service has been paused .
```

Comment:
Please elaborate on what pausing means in this context, and why you would want to do that.